sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -267,8 +267,8 @@ Azure SDK for C++ is licensed under the [MIT](https://github.com/Azure/azure-sdk
 [source]: https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/eventhubs
 [azure_identity_pkg]: https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-identity/latest/index.html
 [default_azure_credential]: https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-identity/latest/index.html#defaultazurecredential
-<!-- TODO: Fix go links to refer to C++ documentation when it is published.-->
-[cppdoc]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs
-[cppdoc_examples]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs#pkg-examples
+
+[cppdoc]: https://azuresdkdocs.blob.core.windows.net/$web/cpp/azure-messaging-eventhubs/latest/index.html
+[cppdoc_examples]: https://github.com/Azure/azure-sdk-for-cpp/tree/main/sdk/eventhubs/azure-messaging-eventhubs/samples
```

Comment:
I think there were other links that point to GoLang as well (might be worth doing a cursory search for `pkg.go.dev`).

Could you fix this too?
https://github.com/Azure/azure-sdk-for-cpp/blob/d5c1e01b8857247d862d11a99f808c430ac22223/sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md?plain=1#L76

Reply:
Just updated.